apps/galaxy/src/modules/auth/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { ErrorMessage } from "@hookform/error-message";
+import useSignIn from "../../../services/signin/useSignIn";
+import { Button, Container, FormLabel, FormControl, Input } from "@cc/ui-chakra";
+
+interface FormData {
+  email: string;
+  password: string;
+}
+
+const SignInForm = () => {
+  const { isLoading, handleSignInSubmit } = useSignIn();
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormData>();
```

Comment:
Turns out I was wrong, if you run the application, you might see an error complaining about changing the controlled state from `null` to an actual value.

Consider providing a defaultValue for each field.

Reply:
Actually, I checked the defaultValue is an empty string rather than null. I think it may be caused by `register` since I didn't find the related document in react-hook-form

Reply:
Ahhhh okay, I'm more used to using `useController` rather than `register`, good to know another difference between these two implementation. 👍